docs/example-scenario/apps/hpc-saas.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Computer-aided engineering through high performance computing on Azure
```

Comment:
Let's work on this title.  Does this actually reflect the article?  Would it be better to call it a "Managed system for Computer Aided Design on Azure"?